src/libraries/System.Private.Xml.Linq/tests/XDocument.Common/BridgeHelpers.cs

```diff
@@ -1005,9 +1060,8 @@ public static void CreateBase64TestFile(string strFileName)
                 WriteToBuffer(ref WTextOnly, ref WTextOnlylen, System.BitConverter.GetBytes(strBase64[i]));
             }
 
-            FilePathUtil.addStream(strFileName, new MemoryStream());
-
-            XmlWriter w = XmlWriter.Create(FilePathUtil.getStream(strFileName));
+            var ms = new MemoryStream();
+            XmlWriter w = XmlWriter.Create(ms);
```

Comment:
I think CloseOutput is false by default: https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.Xml/src/System/Xml/Core/XmlWriterSettings.cs#L505

```csharp
using System.Xml;
Console.WriteLine("Hello, World!");
var x = new XmlWriterSettings();
Console.WriteLine(x.CloseOutput); // => false
```

I think the only case this is true when you pass in a file name rather than Stream

Reply:
Ah yes - then my change was unnecessary too.